workshops/models.py

```diff
@@ -329,6 +336,34 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('event_details', args=[self.get_ident()])
 
+    def get_repository_url(self):
+        """Return self.url formatted as it was repository URL.
+
+        Repository URL is: https://github.com/user/repository/
+        If it doesn't match, the original URL is returned."""
+        try:
+            mo = self.WEBSITE_REGEX.match(self.url)
+            return self.REPO_SCHEMA.format(**mo.groupdict())
+        except (TypeError, AttributeError, KeyError):
+            # TypeError: self.url is None
+            # AttributeError: mo is None (ie. it didn't match)
+            # KeyError: mo.groupdict doesn't supply required names to format
+            return self.url
+
+    def get_website_url(self):
+        """Return self.url formatted as it was website URL.
+
+        Website URL is: https://user.github.io/repository/
```

Comment:
"Website URL is as specified in WEBSITE_FORMAT" (again, to avoid doc string going stale).